DeepCat.js

```diff
@@ -49,10 +73,37 @@
 			}
 		} );
 
-		refreshSearchTermMock();
+		if(refreshSearchTermMock())
+			showHint();
 		checkErrorMessage();
 	} );
 
+	function showHint() {
+		if( !mw.cookie.get( makeHintboxCookieToken(mw.msg('hintbox-text'))) ) {
+			var parent= document.getElementById('mw-content-text');
+			var sresults= document.getElementsByClassName('searchresults')[0];
+			var d= parent.insertBefore(document.createElement('div'), sresults);
+			d.style.marginTop= "1em";
+			d.style.marginBottom= "1em";
+			d.innerHTML=
```

Comment:
Interesting. I'm not really a web developer of course, but [this reference](http://www.w3schools.com/jsref/prop_html_innerhtml.asp) for instance says innerHTML is supported by all major browsers. Apparently it is a [standard](https://dvcs.w3.org/hg/innerhtml/raw-file/tip/index.html) property. But after some more research I found [this](http://www.quirksmode.org/dom/html/) which lists some problems in IE which may or may not affect us here. Looking at the same page, "insertAdjacentHTML" seems to be a much better option. [Here](http://stackoverflow.com/questions/3563107/jquery-html-vs-innerhtml) is some discussion on jQuery's html(), including some posts by people having trouble with it. Do you think there's any problem with just using insertAdjacentHTML?